docs/src/changelog.rst.txt

```diff
@@ -150,6 +150,9 @@ Changelog
    * - ``24.2.0``
      - 11/11/21
      - Added ``filter_mode`` to :carta:ref:`FileListRequest`, :carta:ref:`CatalogListRequest` and :carta:ref:`RegionListRequest` messages; Added ``Unknown`` to :carta:ref:`CatalogFileType`.
+   * - ``24.3.0``
```

Comment:
I guess technically the changes in this PR are non-breaking, but I think we should also bump the ICD version (and thus the major version of the protobufs) when we introduce major new sets of messages like this. There's precedent for this (e.g. the concatenated messages above)